public/Script.No1/count.js

```diff
@@ -0,0 +1,13 @@
+"use.strict";
```

Comment:
strictモードってピリオド不要な気がします！
```suggestion
"use strict";
```

Reply:
これは意図的についたやつ？？